modules/odc-connecting-components.adoc

```diff
@@ -61,36 +61,39 @@ Currently, a few specific Operators like the *etcd* and the *PostgresSQL Databas
 
 You can establish a binding connection with Operator-backed components.
 
-This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Node.js application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub* using a backing `OperatorSource`, and then install the Operator.
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Node.js application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub* using a backing `CatalogSource` resource, and then install the Operator.
```

Comment:
@pmacik thank you for your further update. I think it might be worth clarifying what the `CatalogSource` resource and the `Database` resource do here. In this line it states that a "backing `CatalogSource` resource" is used, but line 71 refers to a "`CatalogSource` resource" and a "backing `Database` resource". I think this introduction would be a good place to explain what the two components enable you to achieve and how they each contribute to that.

You could potentially consider moving the conceptual description in line 71 to this intro and leave the procedure to just state the actions.

Reply:
@pneedle-rh I've updated the docs accordingly with @Preeticp's help.